build.xml

```diff
@@ -148,7 +148,9 @@
 
   <target name="run" depends="jar"
           description="Runs the development version of the program">
```

Comment:
This description is no longer appropriate to this task. Please revise to match.

Reply:
Description still needs to be changed, so that it doesn't claim to run the development version of the program.

Reply:
All requested changes have been made, please review